docs/libcudacxx/standard_api/numerics_library/linalg.rst

```diff
@@ -0,0 +1,28 @@
+.. _libcudacxx-standard-api-numerics-linalg:
+
+``<cuda/std/__linalg/scaled>``
+``<cuda/std/__linalg/conjugated>``
+``<cuda/std/__linalg/transposed>``
+``<cuda/std/__linalg/conjugate_transposed>``
+============================================
+
+Extensions
+----------
+
+-  C++26 ``std::linalg`` accessors, transposed layout, and related functions are available in C++17
+
+   - ``scaled()`` and ``scaled_accessor``
+   - ``conjugated()`` and ``conjugated_accessor``
+   - ``transposed()`` and ``layout_transpose``
+   - ``conjugate_transposed()``
```

Comment:
This should be in the section above, we only need to mention that we backport the features to C++17